raiden/blockchain/events.py

```diff
@@ -144,7 +147,7 @@ def get_all_netting_channel_events(
         pyethapp_chain,
         netting_channel_address,
         events=ALL_EVENTS,
-        from_block=0,
+        from_block='earliest',
```

Comment:
Maybe, they copied `toBlock` documentation in `fromBlock`, and didn't update it. Empirically, `fromBlock='earliest'` returns all events registered in the blockchain, while the current version (without this PR), has `latest` as default, and returns only events from current head.

Looks like the correct behavior for `fromBlock` is something like:
- `earliest`: from oldest block, i.e. all registered events since beggining of time
- `latest`: from most recent mined block
- `pending`: transactions not yet mined ( `> latest`)